distributed/stealing.py

```diff
@@ -45,7 +45,8 @@ def __init__(self, scheduler):
             self.add_worker(worker=worker)
 
         pc = PeriodicCallback(callback=self.balance,
-                              callback_time=100)
+                              callback_time=100,
+                              io_loop=self.scheduler.loop)
```

Comment:
Am surprised this works with Tornado 5.0. Looks like they dropped the [`io_loop` argument to `PeriodicCallback`]( http://www.tornadoweb.org/en/branch5.0/ioloop.html#tornado.ioloop.PeriodicCallback ).

Having a similar discussion about problems this causes for Tornado 5.0 usage in PR ( https://github.com/dask/dask-drmaa/pull/61 ).

Reply:
from distributed.utils import PeriodicCallback

Reply:
Got it. Thanks. :)